zulip_bots/zulip_bots/test_finder.py

```diff
@@ -18,3 +21,20 @@ def test_resolve_bot_path_for_custom_bot(self):
                 expected_bot_path_and_name = ('path/to/custom_handler.py', 'custom_handler')
                 bot_path_and_name = finder.resolve_bot_path(custom_bot_handler)
                 self.assertEqual(bot_path_and_name, expected_bot_path_and_name)
+
+    def test_resolve_bot_path_for_existing_bot(self) -> None:
+
+        def abspath_side_effect(path: Text) -> Text:
+            return 'absolute/path/to/'+path
+
+        with patch('os.path.isfile', return_value=False), \
+                patch('os.path.dirname'), \
+                patch('os.path.abspath', side_effect=abspath_side_effect):
+
+            expected_existing_bot_path_and_name = ('absolute/path/to/api-repo/zulip_bots/zulip_bots/bots/helloworld/helloworld.py',
+                                                   'helloworld')
+            existing_bot_path_and_name = finder.resolve_bot_path('helloworld')
+            self.assertEqual(existing_bot_path_and_name, expected_existing_bot_path_and_name)
```

Comment:
From how I see it, this function doesn't really test resolve_bot_path for an existing bot, since the paths used are still nonexistent. I'd just make a single function `test_resolve_bot_path` that tests to resolve the actual path for an actual bot.